src/rosdiscover/recover/loader.py

```diff
@@ -81,6 +92,10 @@ def _load_float_literal(self, dict_: t.Mapping[str, t.Any]) -> FloatLiteral:
         assert dict_["kind"] == "float-literal"
         return FloatLiteral(value=float(dict_["literal"]))
 
+    def _load_int_literal(self, dict_: t.Mapping[str, t.Any]) -> IntLiteral:
+        assert dict_["kind"] == "int-literal"
+        return IntLiteral(value=int(dict_["literal"]))
```

Comment:
you shouldn't need to have this `int(...)` conversion here

Reply:
removed